utilities/getPageMetadata.ts

```diff
@@ -70,7 +70,7 @@ export const getPageMetadata = ({
   }
 
   // Self reference URL is the page's URL without any query params
-  const selfReferencingUrl = `${siteUrl}/${slug}`;
+  const selfReferencingUrl = slug !== 'home' ? `${siteUrl}/${slug}` : siteUrl;
```

Comment:
Fixing the canonical URL of the homepage to remove the `home` slug

Reply:
Since I pulled the home page into the root `page.tsx` file you shouldn't have to do this. I'd just leave it alone IMHO.

Reply:
Ah, on second thought, with the canonical URL it doesn't hurt.